src/site/xdoc/server/config-spring-jpa-postgres.xml

```diff
@@ -0,0 +1,105 @@
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing,
+  software distributed under the License is distributed on an
+  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+  KIND, either express or implied.  See the License for the
+  specific language governing permissions and limitations
+  under the License.    
+-->
+<document>
+
+ <properties>
+  <title>Apache James Server 3 - Spring JPA With Postgres Configuration</title>
+ </properties>
+
+<body>
+
+
+    <section name="Introduction">
+      <p>Consult <a href="http://james.apache.org/server/config-system.html">System Configuration</a> to get template, some examples and hints for the configuration with JPA.</p>
+    </section>
+
+    <section name="Current Supported Sequence Database">
+      <p>James supports DERBY as a default database, you do not need to do any additional configuration if you are using Derby</p>
+    </section>
+
+    <section name="Spring JPA With Postgres Configuration">
+      <p>If you wish to use POSTGRES as a database, please follow those steps to configure it</p>
+      <subsection name="Step 1: Create database on Postgres">
+
+        <p>You need to install Postgres or run Postgres with Docker</p>
+
+        <p>Example to install it on Debian:</p>
+        <source>
+          sudo apt-get update && sudo apt-get install postgresql-9.4 postgresql-client-9.4
+        </source>
+        <p>Example to pull and launch Postgres on docker</p>
+        <source>
+          docker run --detach --name postgres library/postgres
+        </source>
+        <p>Postgres does not create database on the fly so you need to create the database manually. Here is the command to perform this action: </p>
+
+        <source>psql --username postgres --tuples-only --command "SELECT 1 FROM pg_database WHERE datname = '<strong><italic>database_name</italic></strong>'" | grep -q 1 || psql --username postgres --command "CREATE DATABASE <strong><italic>database_name</italic></strong>"</source>
+
+        <dl>
+          <dt><strong>database_name</strong></dt>
+          <dd>The name of database</dd>
+        </dl>
+
+      </subsection>
+
+      <subsection name="Step 2: Point to the database in configuration file">
+        <p>Change the content of <italic>james-database.properties</italic> and point to the 'database_name' created earlier</p>
+        <source>
+          database.driverClassName=org.postgresql.Driver
+          database.url=jdbc:postgresql://<strong>ip_of_postgres</strong>/<strong>database_name</strong>
+          database.username=postgres
+          database.password=postgres
+
+          vendorAdapter.database=POSTGRESQL
+        </source>
+        <p>If you are using the James Spring docker image. You need to customise the configuration file of the docker container with those steps:</p>
+          <ul>
+            <li>Create a new james-database.properties file with above content</li>
```

Comment:
```
<li>Copy the entire container configuration in a local folder. It can be found <a href="">here</a>.</li>
<li>Update the james-database.properties as stated above</li>
<li>Then customize the configuration file of docker container throw volume</li>
```

We can not expose partial documentation on the volume...